contrib/kafka/cmd/receive_adapter/main.go

```diff
@@ -95,7 +95,7 @@ func main() {
 	stopCh := signals.SetupSignalHandler()
 
 	logger.Info("Starting Apache Kafka Receive Adapter...",
-		zap.String("bootstrapServer", adapter.BootstrapServers),
+		zap.String("bootstrap_server", adapter.BootstrapServers),
 		zap.String("Topics", adapter.Topics),
 		zap.String("ConsumerGroup", adapter.ConsumerGroup),
```

Comment:
Similarly I would keep the names of parameters exactly the same as far as capitalization goes: `topics`, `consumerGroup` etc.

```
		zap.String("consumerGroup", adapter.ConsumerGroup),
```

Reply:
This was already discussed: https://github.com/knative/eventing-sources/pull/329#discussion_r271031783